trillian.proto

```diff
@@ -99,15 +99,15 @@ enum TreeType {
   // Tree represents a verifiable log.
   LOG = 1;
 
-  // Tree represents a verifiable map.
-  MAP = 2;
+  // Deprecated: verifiable maps are no longer supported.
+  DEPRECATED_MAP = 2 [deprecated = true];
```

Comment:
Not sure there's any point deprecating it as it already no longer works. Just reserve the field?

Reply:
Good call, done.